templates/kotsadm-deployment.yaml

```diff
@@ -25,6 +25,12 @@ spec:
       {{- end }}
       containers:
       - env:
+        - name: REPLICATED_APP_ENDPOINT
```

Comment:
nit: would suggest moving these down below next to the other envs we configure and keep `extraEnv` at the beginning.

Reply:
it felt like having this before is more desirable so that extraEnv would take precedence over the hardcoded ones

Reply:
ah makes sense.